src/kubectl-object-model.ts

```diff
@@ -21,7 +23,13 @@ export class Kubectl {
     }
 
     public apply(configurationPaths: string | string[]) {
-        return this.execute(['apply', '-f', this.createInlineArray(configurationPaths)]);
+        if (this.forceDeployment) {
+            console.log("force flag is on, deployment will continue even if previous deployment already exists");
+            return this.execute(['apply', '--force', '-f', this.createInlineArray(configurationPaths)]);
+        }
+        else {
+            return this.execute(['apply', '-f', this.createInlineArray(configurationPaths)]);
+        }

```

Comment:
Call execute from one place.

Do something like
```
initiate array
if(force) add --force to array
array.append(more args)

this.execute()
```

Don't you think this looks cleaner